pkg/liqo-controller-manager/resource-request-controller/testutils_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package testutils
+package resourcerequestoperator
```

Comment:
Wondering why this change is required. If desired, would postpone to a later PR, to keep this one simple (given its criticality).

Reply:
Because the testutils now depend on the "parent" package: `checkResourceOfferUpdate` calls `ScaleResources`. Of course, one could just move `checkResourceOfferUpdate` to `resourceRequest_operator_test` and keep the testutils package - let me know which option would be better.

Reply:
OK, got it. Overall the entire test suite of the resource request controller seems strange to me (these methods included), but that is a separate matter.